src/chapters/02_docker/03_running-containers/index.md

```diff
@@ -116,6 +116,21 @@ When inspecting the `wordpress-network` using `docker network inspect wordpress-
 ]
 ```
 
+### Creating a database
```

Comment:
this should not be necessary, the system needs to be deployable without any interaction (only some docker commands)